basis/vocabs/refresh/refresh.factor

```diff
@@ -1,8 +1,8 @@
 ! Copyright (C) 2009 Slava Pestov.
 ! See http://factorcode.org/license.txt for BSD license.
-USING: accessors assocs checksums checksums.crc32
-combinators.short-circuit io.encodings.utf8 io.files kernel
-namespaces sequences sets source-files vocabs vocabs.loader ;
+USING: accessors assocs checksums checksums.crc32 io.encodings.utf8 io.files
+kernel namespaces sequences sets source-files tools.test.private vocabs
```

Comment:
I'm not sure vocabs.refresh can depend on tools.test.

It is likely that ui.tools can depend on tools.test if you want to move these words to tools.test.

Reply:
> I'm not sure vocabs.refresh can depend on tools.test.
> 
> It is likely that ui.tools can depend on tools.test

I tried boostrapping, and everything seemed to be fine.  Is there another reason why it should not depend on it?
I can move these words to `tools.test` nevertheless.

Reply:
Right now both tools.test and vocabs.refresh are part of "bootstrap.tools", so that might currently be the case, but I'm not sure but it seems tools.test is the more optional vocabulary.

Reply:
I moved the code over to `tools.test`.  Please have a look if it looks correct.